internal/pkg/manifest/job.go

```diff
@@ -36,6 +36,7 @@ type ScheduledJob struct {
 
 // ScheduledJobConfig holds the configuration for a scheduled job
 type ScheduledJobConfig struct {
+	name                    string
```

Comment:
Are we setting these values anywhere?

Reply:
Not yet until we actually use this new manifest validation